site/content/docs/5.3/helpers/icon-link.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+title: Icon link
+description: Quickly create stylized hyperlinks with Bootstrap Icons or other icons.
```

Comment:
Solaris icons ?

Reply:
Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1854/commits/0f7c7fa8558396f71d3424b13ac8a5dc03ad63e6